src/main/java/kg/jarkyn/server/Outgoing/Responder.java

```diff
@@ -0,0 +1,8 @@
+package kg.jarkyn.server.Outgoing;
+
+import kg.jarkyn.server.Incoming.Requester;
+
+public interface Responder {
```

Comment:
i actually didn't want the interface, it somehow emerged from tests, must got carried away, should have just extended the class for testing, will have a look at this